resources/lib/objects/kodi/music.py

```diff
@@ -189,14 +189,15 @@ def add_genres(self, kodi_id, genres, media):
 
         ''' Add genres, but delete current genres first.
         '''
+		#Removed for DB version 72
         if media == 'album':
-            self.cursor.execute(QU.delete_genres_album, (kodi_id,))
-
+            #self.cursor.execute(QU.delete_genres_album, (kodi_id,))
```

Comment:
can you explain the change?

Reply:
Can’t seem to find a substitute for the “genres” table in the new db, seems they store it in the songs table.
Keeping it in the code for reference